elements/rh-accordion/rh-accordion.css

```diff
@@ -5,11 +5,10 @@
   background-color: var(--rh-color-white, #ffffff);
 }
 
-#container { display: contents; }
-
-.dark {
-  --_border-color: var(--rh-color-border-subtle-on-light, #d2d2d2);
+:host([on="dark"]) {
```

Comment:
I've updated this in the PR along with the attribute false, not sure what we were seeing earlier today where the attribute false ended up fixing the styling on the child components because I cannot get that fix to work anymore.  It looks like the color context sets the dark property and then overwrites it right afterwards with a null property

Reply:
I'll take a look

Reply:
sent some commits. needs a bit of polish on the typescript side, but runtime I *think* is correct.

Reply:
[Context should be g2g now](https://deploy-preview-594--red-hat-design-system.netlify.app/components/accordion/demo/color-context/)